03LessonThree/homework/git-scm.html

```diff
@@ -12,14 +12,14 @@
             <span><a href="#">Home</a></span>
             <span><a href="#">How does it work</a></span>
             <span><a href="#">Contact Us</a></span>
-        </div>
-    </div>
+        </nav>
+    </header>
     <main>
         <section>
             <img src="http://git-scm.com//images/logo@2x.png" alt="git logo"/>
             <h2>Welcome to Git</h2>
             <p>Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency. </p>
-        </div>
+        </section>
         <section>
             <article>
                 <h3>About</h3>

```

Comment:
Still missing some of the remaining closing tags. Make sure to check the closing tags for the articles, asides and the footer.